airflow/providers/amazon/aws/hooks/athena.py

```diff
@@ -188,17 +189,35 @@ def get_query_results_paginator(
         paginator = self.get_conn().get_paginator('get_query_results')
         return paginator.paginate(**result_params)
 
-    def poll_query_status(self, query_execution_id: str, max_tries: Optional[int] = None) -> Optional[str]:
+    def poll_query_status(
+        self,
+        query_execution_id: str,
+        max_polling_attempts: Optional[int] = None,
+        max_tries: Optional[int] = None,
+    ) -> Optional[str]:
         """
         Poll the status of submitted athena query until query state reaches final state.
         Returns one of the final states
 
         :param query_execution_id: Id of submitted athena query
-        :param max_tries: Number of times to poll for query state before function exits
+        :param max_tries: Deprecated - Use max_polling_attempts instead
+        :param max_polling_attempts: Number of times to poll for query state before function exits
         :return: str
         """
+        if max_tries:
+            warnings.warn(
+                f"Method `{self.__class__.__name__}.max_tries` is deprecated and will be removed "
+                "in a future release.  Please use method `max_polling_attempts` instead.",
```

Comment:
This message does not read right. [Method](https://en.wikipedia.org/wiki/Method_(computer_programming)) is a function on an instance, but these are not.

Reply:
Missed that, thanks!